src/diffusers/loaders/lora.py

```diff
@@ -625,9 +519,7 @@ def load_lora_into_text_encoder(
                 # Unsafe code />
 
     @classmethod
-    def load_lora_into_transformer(
-        cls, state_dict, network_alphas, transformer, low_cpu_mem_usage=None, adapter_name=None, _pipeline=None
```

Comment:
We have absolutely no reason to continue to include `low_cpu_mem_usage` when using `peft`.